_posts/2024-02-26-cache-refresh.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title:  "A cache refresh for Trino"
+author: Manfred Moser
+image: /assets/blog/trino-cache-refresh.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+Leon Bambrick once said "There are only two hard things in computer science:
+cache invalidation, naming things, and off-by-one errors" improving the initial
+wisdom from [Phil Karlton](https://www.karlton.org/2017/12/naming-things-hard/).
+Well, in the Trino community we know all about caching, naming, off-by-one
+errors, and improving on original wisdom. With the recent [Trino 439
+release]({{site.url}}/docs/current/release/release-438.html) caching from object
+storage file systems got a refresh. Users of catalogs using the Delta Lake,
+Hive, and Iceberg connectors now get to enjoy performance benefits from the new
+Alluxio-powered file system caching. And all that as a replacement from the now
+removed object storage caching with RubiX.
+
+<!--more-->
+
+## In the past
+
+So how did we get here? A long, long time ago Quoble open sourced a [light
+light-weight data caching framework called
+RubiX](https://github.com/qubole/rubix). The library was integrated into the
+Trino Hive connector and enabled [Hive connector storage
+caching]({{site.url}}/docs/438/connector/hive-caching.html). However, over time
```

Comment:
We need a comma after "over time," so let's change "However" to "But" to simplify the flow and not have too many commas breaking things up.
```suggestion
caching]({{site.url}}/docs/438/connector/hive-caching.html). But over time,
```

Reply:
Done